docs/variables/tiktok.md

```diff
@@ -0,0 +1,91 @@
+---
+id: tiktok
+---
+
+# $(tiktok)
+
+Returns a range of metadata around a broadcaster's latest [**TikTok**](https://www.tiktok.com/en/) video that can be accessed via multiple different placeholders.
```

Comment:
Should be a note here that the broadcaster must authorise TikTok on their integrations page, rather than down below in the error, maybe an an info box or something